### ISSUE TITLE
invalid operands for multiplication

### ISSUE THREAD

**Issue by mohamnag** [atomicpages/skeleton-sass#16]
it seems that this code does not get compiled with node-sass@2.0.1
what I have done is:
```
> npm install node-sass -g
...
node-sass@2.0.1 install

> bower install skeleton-sass
....
bower install       skeleton-sass#2.0.4

> cd bower_components/skeleton-sass/
```
now running the following command:
```
node-sass skeleton_template.scss
```
will return:
```
{
  "status": 1,
  "file": "/Users/mohamnag/Projects/experiments/test-skscss/bower_components/skeleton-sass/skeleton/themes/sphenoid/_base.scss",
  "line": 203,
  "column": 11,
  "message": "invalid operands for multiplication",
  "code": 1
}
```
tracing the code shows, it has nothing with that line to do, but actually with line 205, from file `bower_components/skeleton-sass/skeleton/core/_dependencies.scss` which is:
```
@return ($px / $base) * 1#{$unit};
```
I have tried it on another machine with clean nodejs installation and I have the same problem, is this a bug in code? or a config problem?

**Comment by atomicpages** [comment]
This seems to be a bug with `libsass`. I've always built around the official gem (which never complained about this) but I'll look into it!

**Comment by mohamnag** [comment]
yes somebody says it is fixed in 3.1 here: https://github.com/sass/libsass/issues/152

I'm not sure which version is node-sass using now, but I wonder how do you manage to compile if thats not working.

**Comment by atomicpages** [comment]
How did you install libsass I installed a fresh copy with [homebrew](http://brew.sh/) `brew install --HEAD libsass`. Also, that fix only is for the mod operator and doesn't account for this specific issue it seems :(

Finally, it appears as though the version of `libsass` that `node-sass` is using is from Jan 1 so it's a little behind schedule. I clone the master branch from the `libsass` repo into my `node-sass` with no avail. I'll see if I can track down the bug in `libsass`. For an immediate fix you can use [the official sass gem](http://sass-lang.com/) to resume your work!

**Comment by mohamnag** [comment]
I actually am not sure how did I install sass, I'm using it on a daily basis and I assume at some point I have installed the libsass using brew. I tried to replace the libsass with the head of repo but even that did not change anything.
I hope this can be fixed soon as I'm looking forward to using this very well modularised version of skeleton with more ease.

**Comment by atomicpages** [comment]
[`node-sass` comes with `libsass`](https://github.com/sass/node-sass/tree/master/src) so it uses the internal version which is why you can use it right out of the box. However, the version that's currently bundled (at least at the time of this writing) is from Jan 1. I noticed this and cloned a fresh copy into the directory where `node-sass` is located but that didn't work. 

I'll try to touch base with the `libsass` community and see if they can help!

**Comment by atomicpages** [comment]
Finally submitted an issue on libsass. You can keep track of the issue here: https://github.com/sass/libsass/issues/948

**Comment by mohamnag** [comment]
great, will keep an eye on that issue too.

**Comment by atomicpages** [comment]
@mohamnag it's been a little while but https://github.com/sass/libsass/issues/948 still hasn't been resolved. I'll try to have a work around sometime relatively soon!

**Comment by atomicpages** [comment]
@mohamnag this has been resolved in the 2.5.1 release thanks to pull #20!
